src/main/java/com/fizzbuzz/problem/FizzBuzzProblem.java

```diff
@@ -1,4 +1,28 @@
 package com.fizzbuzz.problem;
 
 public class FizzBuzzProblem {
+
+    public static void main(String[] args) {
+    FizzBuzzProblem fizzBuzzProblem = new FizzBuzzProblem();
+
+
+    fizzBuzzProblem.getFizzBuzz(15);
+
+    fizzBuzzProblem.getFizzBuzz(9);
+
+    fizzBuzzProblem.getFizzBuzz(10);
+
+    fizzBuzzProblem.getFizzBuzz(17);
+    }
+
+    public String getFizzBuzz(int number){
+        StringBuilder returnString = new StringBuilder("");
```

Comment:
would have preferred simple, straight forward approach which is more readable and easier to understand